rllib/policy/torch_policy_v2.py

```diff
@@ -237,7 +237,19 @@ def loss(
         Returns:
             Loss tensor given the input batch.
         """
-        raise NotImplementedError
+        # Under the new _enable_learner_api the loss function still gets called in order
+        # to initialize the view requirements of the sample batches that are returned by
+        # the sampler. In this case, we don't actually want to compute any loss, however
+        # if we access the keys that are needed for a forward_train pass, then the
+        # sampler will include those keys in the sample batches it returns. This means
+        # that the correct sample batch keys will be available when using the learner
+        # group API.
+        if self.config._enable_learner_api:
+            for k in model.input_specs_train():
+                train_batch[k]
+            return torch.Tensor([0.0])
```

Comment:
why return 0.0 tensor? why not return None?

Reply:
I did some more digging, and we can choose to return none over here since the loss is not used in any way when initializing view requirements. 

I'll go ahead and return instead None type.

Reply:
fixed.